examples/wrapper-components/react-typescript/src/App.tsx

```diff
@@ -44,13 +48,17 @@ function App() {
          <Accordion />
          <br />
 
-      <h2>Radio Button</h2>
-      <RadioButton />
-      <br />
+         <h2>Radio Button</h2>
+         <RadioButton />
+         <br />
+
+         <h2>Tabs</h2>
+         <Tabs />
+         <br/>
 
-      <h2>Tabs</h2>
-      <Tabs />
-      <br/>
+         <h2>Tabs</h2>
+         <Card />
+         <br/>
```

Comment:
should be Card, that headline is what i meant

Reply:
I know. I already noticed it, and changed it :D